screenpipe-audio/Cargo.toml

```diff
@@ -80,10 +80,12 @@ criterion = { workspace = true }
 memory-stats = "1.0"
 
 [features]
+default = ["metal"]
```

Comment:
why metal? and remove cuda? ???


windows / linux user dont want to use metal